python/paddle/fluid/tests/unittests/test_amp_check_finite_and_scale_op.py

```diff
@@ -0,0 +1,97 @@
+#   Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+import numpy as np
+from op_test import OpTest, skip_check_grad_ci
+import paddle.fluid as fluid
+
+
+@skip_check_grad_ci(
+    reason="Operator amp_check_finite_and_scale is used for dygraph auto-mixed_precision training only. It has no grad op."
+)
+class TestAmpCheckFiniteAndScaleOp(OpTest):
+    def setUp(self):
+        self.op_type = "amp_check_finite_and_scale"
+        self.init_dtype()
+        x = np.random.random((1024, 1024)).astype(self.dtype)
+        scale = np.random.random((1)).astype(self.dtype)
+
+        self.inputs = {'X': [('x0', x)], 'Scale': scale}
+        self.outputs = {
+            'FoundInfinite': np.array([0]),
+            'Out': [('out0', x * scale)],
+        }
+
+    def init_dtype(self):
+        self.dtype = np.float32
+
+    def test_check_output(self):
+        self.check_output()
+
+
+@skip_check_grad_ci(
+    reason="Operator amp_check_finite_and_scale is used for dygraph auto-mixed_precision training only. It has no grad op."
+)
```

Comment:
First it will be checked if there is a grad op. If no grad op, the unittest will not be required checking grad. So there is no need to use skip_check_grad_ci.

Reply:
Done, thanks.